tests/installation/select_patterns_and_packages.pm

```diff
@@ -69,6 +69,10 @@ sub gotopatterns {
         $self->record_dependency_issues;
     }
 
+    if (check_screen('dependancy-issue', 10) && get_var("BREAK_DEPS")) {
```

Comment:
try to avoid timeout values which are neither 0 nor bigger than the default because they tend to be not robust, e.g. when the machine is a bit loaded. It should work with argument 0, especially when combined with synchronization methods used in before, e.g. `wait_screen_change` or `wait_still_screen`

This comment also applies to all other check_screens you introduced